[2023-12-26T22:55:23.997+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: KOH.get_data manual__2023-12-26T17:55:20.000429+00:00 [queued]>
[2023-12-26T22:55:24.004+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: KOH.get_data manual__2023-12-26T17:55:20.000429+00:00 [queued]>
[2023-12-26T22:55:24.005+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-26T22:55:24.040+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): get_data> on 2023-12-26 17:55:20.000429+00:00
[2023-12-26T22:55:24.044+0500] {standard_task_runner.py:57} INFO - Started process 11552 to run task
[2023-12-26T22:55:24.077+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'KOH', 'get_data', 'manual__2023-12-26T17:55:20.000429+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/text.py', '--cfg-path', '/tmp/tmphom1vmgn']
[2023-12-26T22:55:24.110+0500] {standard_task_runner.py:85} INFO - Job 58: Subtask get_data
[2023-12-26T22:55:24.669+0500] {task_command.py:416} INFO - Running <TaskInstance: KOH.get_data manual__2023-12-26T17:55:20.000429+00:00 [running]> on host vm2.myguest.virtualbox.org
[2023-12-26T22:55:24.844+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Admin' AIRFLOW_CTX_DAG_ID='KOH' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-26T17:55:20.000429+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-26T17:55:20.000429+00:00'
[2023-12-26T22:55:24.845+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-26T22:55:24.848+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/kda/ML_lab3/MLops_lab3/datasets && gdown 1X-mGhWO8DwPLljP1e0gkdAloD-GL8SLq']
[2023-12-26T22:55:24.868+0500] {subprocess.py:86} INFO - Output:
[2023-12-26T22:55:27.506+0500] {subprocess.py:93} INFO - Downloading...
[2023-12-26T22:55:27.506+0500] {subprocess.py:93} INFO - From: https://drive.google.com/uc?id=1X-mGhWO8DwPLljP1e0gkdAloD-GL8SLq
[2023-12-26T22:55:27.507+0500] {subprocess.py:93} INFO - To: /home/kda/ML_lab3/MLops_lab3/datasets/Sport_car_price.csv
[2023-12-26T22:55:27.540+0500] {subprocess.py:93} INFO -   0%|          | 0.00/49.2k [00:00<?, ?B/s]100%|##########| 49.2k/49.2k [00:00<00:00, 50.5MB/s]
[2023-12-26T22:55:27.630+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-26T22:55:27.692+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=KOH, task_id=get_data, execution_date=20231226T175520, start_date=20231226T175523, end_date=20231226T175527
[2023-12-26T22:55:27.761+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-26T22:55:27.879+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
